### ISSUE TITLE
[enhancement] consider album/CD number for storing

### ISSUE THREAD

**Issue by mojo2012** [vincelwt/harmony#212]
I have some albums on google music that are composed of 2 CDs. The songs have the same track numbers. This causes the sorting to be wrong: 1, 1, 2, 2, 3, 3 ...
Sorting should consider the CD number too, if available.

**Comment by vincelwt** [comment]
Agree, I thought it was already like this, it's not working properly. I'll fix it

**Comment by vincelwt** [comment]
Alright should use the proper sorting with next release :)
